src/screens/User.jsx

```diff
@@ -11,9 +12,10 @@ const User = ({ cname }) => {
 
   return (
     <section className={cname}>
-      {authState.user && (
+      {authState ? (
         <p className={styles.greet}>Hello, {authState.user.username}👋</p>
-      )}
+      ) : null}
```

Comment:
you are checking `authState` an object here, since its a private route I do not think you need this check. If it is needed you can check `authState.token` or `authState.user`

Reply:
Thanks for pointing it out. It's fixed now, I initially changed it to debug something but forgot to change it back.